extensions/wikia/ContentReview/controllers/ContentReviewSpecialController.class.php

```diff
@@ -36,6 +36,8 @@ public function index() {
 			return false;
 		}
 
+		$this->wg->Out->setPageTitle( wfMessage( 'content-review-special-title' )->escaped() );
```

Comment:
`$this->getOutput()` is preferred over accessing the global when it's available, which it is in special pages. Also, `OutputPage::setPageTitle` sanitizes and escapes its input, so no need to use `->escaped()`.